android/app/src/main/java/org/chrome_on_device_ml/ml/QaClient.java

```diff
@@ -0,0 +1,228 @@
+/* Copyright 2020. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+package org.chrome_on_device_ml.ml;
+
+import android.content.Context;
+import android.content.res.AssetFileDescriptor;
+import android.content.res.AssetManager;
+import androidx.annotation.WorkerThread;
+import android.util.Log;
+import com.google.common.base.Joiner;
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.ByteBuffer;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.tensorflow.lite.Interpreter;
+
+/** Interface to load TfLite model and provide predictions. */
+public class QaClient implements AutoCloseable {
+  private static final String TAG = "CDML_QaClient";
+  private static final String MODEL_PATH = "bert_model.tflite";
+  private static final String DIC_PATH = "bert_vocab.txt";
+
+  private static final int MAX_ANS_LEN = 32;
+  private static final int MAX_QUERY_LEN = 64;
+  private static final int MAX_SEQ_LEN = 384;
+  private static final boolean DO_LOWER_CASE = true;
+  private static final int PREDICT_ANS_NUM = 5;
```

Comment:
NUM_PREDICT_ANS?

Reply:
This is copied from TF example. It's the X number of top answers.

Reply:
Add comment please

Reply:
Done.